client/packages/common/src/ui/components/inputs/DatePickers/BaseDatePickerInput/BaseDatePickerInput.tsx

```diff
@@ -50,7 +55,9 @@ export const BaseDatePickerInput: FC<
   const t = useTranslation('common');
 
   const dateValue =
-    typeof props.value === 'string' ? new Date(props.value) : props.value;
+    typeof props.value === 'string'
```

Comment:
oops. this is an offroad! I fixed #2153 because it was causing an issue while testing a different issue.